casper/src/main/scala/coop/rchain/casper/engine/Running.scala

```diff
@@ -30,6 +29,7 @@ object Running {
   final case class Requested(
       timestamp: Long,
       peers: Set[PeerNode] = Set.empty,
+      received: Boolean,
```

Comment:
It makes sense to set default _false_ for received field.

Reply:
https://github.com/rchain/rchain/pull/2902/commits/b94764d552d88a30e0bea77be8c951688d9024ca